examples/feeds/link_feed_item_set.rb

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env ruby
+# Encoding: utf-8
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Links the specified feed item set to the specified feed item. The specified
+# feed item set must not be created as a dynamic set, i.e. both
+# FeedItemSet::dynamic_location_set_filter and
```

Comment:
`FeedItemSet::dynamic_location_set_filter` -> Is this the normal format of Ruby? If not, could you also change?